lib/rules/web/admin_console/4.12/baremetalhost.xyaml

```diff
@@ -0,0 +1,13 @@
+check_baremetalhost_status_popup:
+  elements:
+  - selector:
+      xpath: (//span[@data-test='status-text'])[1]
```

Comment:
to make it more stable, maybe we should update to? 
```
xpath: (//span[@data-test='status-text' and text()='Externally provisioned'])[1]
```
![Screenshot 2023-04-19 at 10 58 09 AM](https://user-images.githubusercontent.com/12692381/232955575-07eac8c4-02e4-4d40-9b51-e25c6527dc5a.png)